README.md

```diff
@@ -1,6 +1,37 @@
-# curriculum-cr-training-mobile-menu
+# Curriculum CR Training Mobile Menu
 
 
 This repo host an initial code for the tasks performed during the Code Reviewers training.
 
 Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+
+## :red_circle: Live Demo
+
+- Under construction
+
+## :hammer: Built With
+
+
+## :blue_book: Learning Objectives
```

Comment:
- [ ] I think it would be better if you update your README file with all the required/needed information. Remember that most of the time recruiters will see the README and this should have all the necessary information for timing and professional reasons. This could be helpful when you're in a hiring process.